src/main/resources/import.sql

```diff
@@ -0,0 +1,46 @@
+INSERT INTO COURSE (COURSE_ID, COURSE_NAME, SEMESTER, STUDENT_COUNT, TIME_UPDATED) VALUES (1, 'SOFTENG 761', 'Semester 1 2022', 9, '2022-03-16 14:29:36');
```

Comment:
Just wondering, do we still need the separate SQL files for each integration test?

Reply:
currently we do still need some of them, since the data I have setup is purely just for mock data in the demo, and it doesn't add buddies or messages etc, a separate issue could be made for unifying the setup SQL files for the integration tests and then changing the tests to work for them.

Another thing is that some of the tests setup the data in a certain way to test edge cases, and this would need to be maintained when moved over. For now I think our efforts are better spent on other issues

Reply:
That makes sense, thanks for the answer :)